docs/user/ensure_your_experiments_are_reproducible.md

```diff
@@ -0,0 +1,100 @@
+
+# Ensure your experiments are reproducible
+
+Ensure the reproducibility of your experiments with comprehensive launcher
+files. Launcher files are used to initialize the algorithm, its components
+(policy, replay buffer, etc.), and tools for managing experiments.
+
+There are some important details in this example launcher file.
+
+```python
+from garage import wrap_experiment
+from garage.envs import GarageEnv
+from garage.experiment import deterministic, LocalRunner
+import a.b.c
+
+@wrap_experiment(wrap_experiment_args, wrap_experiment_kwargs)
+def experiment(ctxt=None, seed):
+    deterministic.set_seed(seed)
+    runner = LocalRunner(snapshot_config=ctxt)
+    env = GarageEnv(YOUR_ENV)
+
+    # Initialize algorithm dependencies (policy, Q-functions,
+    # replay buffers, etc.)
+    policy = ...
+    algo = ...
+
+    runner.setup(algo=algo, env=env)
+    runner.train(n_epochs=num_epochs, batch_size=)
+
+seed = 0
+experiment(seed)
+```
+
+## Important Take Aways
+
+### Use `garage.wrap_experiment` for snapshotting and saving results
+
+Use the function decorator `wrap_experiment`  to enable the storage of results
+of your experiment. By default, experiments are named based on the name of the
+function that is wrapped by the wrap experiment decorator. In the case of the
+example above, experiments would be saved under the directory name
+`experiment/`. You can specify the directory in which all results are saved to
+using the parameter `log_dir`. You can further modify the directory tree under
+which your experiments are saved and the experiment naming notation by using
+the parameters `prefix`, `name` , `use_existing_dir` and `name_parameters`.
+For example, by setting `name_parameters` to true, the parameters to the
+function wrapped by `wrap_experiment` will be incorporated into the directory
```

Comment:
`@wrap_experiment` has been covered in other docs. I would suggest only focussing on "reproduce experiment"-related topics here. For other info about `@wrap_experiment`, you can do `Find more about how to use @wrap_experiment [here](experiments).`

In other words, this part can only mention the importance of passing in important hyperparameters, instead of giving a complete documentation of these params.